<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportReport_Dialog</class>
 <widget class="QDialog" name="ExportReport_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>220</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Product Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Product ID:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Quality Tester:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Comments:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Company Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Plot Type:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_1">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>180</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>180</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>180</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>180</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_5">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>130</y>
     <width>180</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_2D">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>160</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>2D</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_3D">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>160</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>3D</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sureBtn">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>190</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Sure</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
